mitodl/mit-open
# Update node and remove docker-compose.override.yml

# What are the relevant tickets?
closes #136 
closes #141 

# Description (What does it do?)
This PR removes docker-compose.override.yml and merges it into `docker-compose.yml` and docker-compose-notebook.yml.


# Screenshots (if appropriate):


# How can this be tested?
1. docker compose up; things should work as normal.
2. Try the notebook: *I wanted to double check this was still working after changes. Actually, it wasn't working before my changes, but now it is.*
    - `cp app.ipynb.example app.ipynb`
    -  `docker compose -f ./docker-compose-notebook.yml up` and open browser to url indicated in logs
    - open app.ipynb and run the top cell
    - run some python

# Additional Context
Based on https://github.com/mitodl/micromasters/pull/3644, I think the docker-compose.override.yml was originally intended to help share settings between local dev docker and travis docker. We don't use travis docker anymore, so it's unnecessary and confusing.
